frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+import * as Urls from "metabase/lib/urls";
+
+import Link from "metabase/collections/components/CollectionSidebar/CollectionSidebarLink";
+import { LabelContainer } from "../Collections/CollectionsList/CollectionsList.styled";
+import BookmarksRoot, {
+  BookmarkLinkRoot,
+  BookmarkTypeIcon,
+} from "./Bookmarks.styled";
+
+import { SidebarHeading } from "metabase/collections/components/CollectionSidebar/CollectionSidebar.styled";
+
+const LabelPropTypes = {
+  itemId: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+};
+
+const BookmarksPropTypes = {
+  bookmarks: PropTypes.object,
+};
+
+const Label = ({ name }) => {
+  return (
+    <LabelContainer>
+      <BookmarkTypeIcon name="grid" />
+      {name}
+    </LabelContainer>
+  );
+};
+
+Label.propTypes = LabelPropTypes;
+
+const CollectionSidebarBookmarks = ({ bookmarks }) => {
```

Comment:
This is a new component, can we implement it using TypeScript? You can use `metabase-types/api` package to store global types, like we do here for timelines https://github.com/metabase/metabase/blob/eb9ef86f68d53374f7d621755ed802c43707027b/frontend/src/metabase-types/api/timeline.ts. Just create a file `bookmark.ts` with type definitions for bookmarks.